arex-instrumentation/redis/arex-redission-v3/src/main/java/io/arex/inst/redisson/v3/RedissonWrapperCommon.java

```diff
@@ -24,10 +25,7 @@ public static <R> RFuture<R> delegateCall(String redisUri, String cmd, String ke
             RedisExtractor extractor = new RedisExtractor(redisUri, cmd, key, field);
             MockResult mockResult = extractor.replay();
             if (mockResult.notIgnoreMockResult()) {
-                if (mockResult.getThrowable() != null) {
-                    return new CompletableFutureWrapper<>(mockResult.getThrowable());
-                }
-                return new CompletableFutureWrapper<>((R) mockResult.getResult());
```

Comment:
Solving compatibility issues is too complicated a solution, how about a custom implementation of an `RFutureWrapper`?